Source/Core/Core/Boot/Boot.h

```diff
@@ -15,6 +15,8 @@
 #include "Common/CommonTypes.h"
 #include "DiscIO/Enums.h"
 #include "DiscIO/Volume.h"
+#include "DiscIO/VolumeFileBlobReader.h"
```

Comment:
What's this doing here?

Reply:
Oops, I had meant to replace this with an `#include "DiscIO/Blob.h"` in Boot.cpp, but it looks like I dropped the fixup commit by mistake... Thanks for catching this, fixed!